doc/source/code_editors.rst

```diff
@@ -1,3 +1,67 @@
 ============
 Code Editors
 ============
+
+==============================
```

Comment:
This is wrong: you make 2 titles of the doc (look at https://thomas-cokelaer.info/tutorials/sphinx/rest_syntax.html#id8 how this work). In principle you only drop upper "=" above `IDE and ...` and that's it

Reply:
Thanks for the input I changed it accordingly